include/command_graph_generator.h

```diff
@@ -218,24 +252,20 @@ class command_graph_generator {
 	 * Generates command(s) that need to be processed by every node in the system,
 	 * because they may require data transfers.
 	 */
-	void generate_distributed_commands(const task& tsk);
+	void generate_distributed_commands(batch& current_batch, const task& tsk);
```

Comment:
Above this point, the name of the `batch` parameter is `batch`, starting from here and below it's `current_batch`. Is there actually a semantic difference or is this arbitrary? (If the latter, I suggest settling on one, preferably `current_batch`)

Reply:
Renamed to `current_batch` everywhere.